### PR TITLE
Adding .adoc files for every .md file

### PR DESCRIPTION
**Pull request by ousleyp**
@fabiand 

Ran pandoc on all .md files under user-guide, transforming them to .adoc files. I have not checked all of the files, but I spot-checked and it appeared promising. Please review when you can.

### CODE REVIEW

File: installation/README.adoc [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+Software emulation
+++++++++++++++++++
+
+If hardware virtualization is not available, then a
+https://github.com/kubevirt/kubevirt/blob/master/docs/software-emulation.md[software
+emulation fallback] can be enabled using:
+
+....
+$ kubectl configmap -n kube-system kubevirt-config \
```
Comment (slintes) [inline installation/README.adoc]: Not sure how we handle this... but fyi: I just opened  a PR on the md file to fix this command to `kubectl create configmap...`

**Comment by fabiand** [comment]
Picking this up now.

The transofrmation is a first step. Now we need to find a replacement for docsify - aka a way to publuish the pages nicely.

**Comment by fabiand** [comment]
@ousleyp once we have fewer than 3PRs then I suggest to redo this conversion and do the switch,

We'll then probably need some time to update a few links (docs.kubevirt.io and kubevirt.io/user-guide)

And we'd need to work on building static pages from the sources

**Comment by ousleyp** [comment]
Ok, that makes sense @fabiand. I'm not sure when I will have the bandwidth
- how urgent is it? Do I need to check the PRs regularly or will you let me
know? Thanks.

Pan Ousley, RHCE
Technical Writer | CNV
Customer Content Services
IRC: pousley | they/them
<https://red.ht/sig>

**Comment by fabiand** [comment]
I think it's not urgent, but I would liek to get to it in the next few
weeks.

**Comment by fabiand** [comment]
Does that reough timeframe work for you (4-6weeks)

**Comment by ousleyp** [comment]
> Does that reough timeframe work for you (4-6weeks)

It sounds good in theory, but I might need some assistance (maybe from Andrew Burden?). I will be out around Thanksgiving and in early December for medical leave.

**Comment by fabiand** [comment]
Closing this in favor of #174

**Pull request closed without merging**